package.json

```diff
@@ -38,20 +38,20 @@
     ]
   },
   "dependencies": {
-    "@jupyterlab/application": "^1.0.0",
-    "@jupyterlab/apputils": "^1.0.0",
-    "@jupyterlab/cells": "^1.0.0",
-    "@jupyterlab/coreutils": "^3.0.0",
-    "@jupyterlab/docmanager": "^1.0.0",
-    "@jupyterlab/docregistry": "^1.0.0",
-    "@jupyterlab/fileeditor": "^1.0.0",
-    "@jupyterlab/markdownviewer": "^1.0.0",
-    "@jupyterlab/notebook": "^1.0.0",
-    "@jupyterlab/rendermime": "^1.0.0",
-    "@phosphor/algorithm": "^1.1.3",
-    "@phosphor/coreutils": "^1.3.1",
-    "@phosphor/messaging": "^1.2.3",
-    "@phosphor/widgets": "^1.8.0",
+    "@jupyterlab/application": "^2.0.0-rc.1",
```

Comment:
Now that JupyterLab 2.0 has been released, can we remove the rc versions here?

Reply:
We should be getting to this in the next day or two, @marthacryan and my team haven't had bandwidth yet since the release.

Reply:
Sure, no rush, I appreciate the effort!